docs/guides/building/arm-cross-compile.md

```diff
@@ -0,0 +1,162 @@
+
+1. Update the distro: `sudo apt-get update && sudo apt-get upgrade -y`
+2. Add Tauri [system requirements].
+3. Install Rust and ensure it's up-to-date: `rustup update`.
+4. Install [Tauri using Cargo]: `cargo install tauri-cli`.
+5. Install necessary tools for your frontend framework, like npm and Next.js.
+
+:::info OpenSSL Workaround for WSL on Windows
+
+If using WSL on Windows and encountering OpenSSL-related errors, add the following line to `Cargo.toml` in `<project-root>/src-tauri/Cargo.toml` within the `[dependencies]` section:
+```toml
+openssl-sys = {version = "0.9.66", features = ["vendored"]}
+```
+:::
+
+### Cross-Compiling
+
+Now, let's cross-compile the Tauri application for ARM:
+
+1. Install Rust target for ARM: `rustup target add armv7-unknown-linux-gnueabihf`.
+2. Install the linker for ARM version: `sudo apt install gcc-arm-linux-gnueabihf`.
+3. Open or create the file `<project-root>/.cargo/config.toml` and add the following:
+   ```toml
+   [target.armv7-unknown-linux-gnueabihf]
+   linker = "arm-linux-gnueabihf-gcc"
+   ```
+4. Enable armhf in the package manager: `sudo dpkg --add-architecture armhf`.
+
+:::info Adjusting Package Sources
+
+On Debian, this step isn't necessary, but on other distributions, you might need to edit `/etc/apt/sources.list` to include the ARM architecture variant. For example, add these lines:
+```bash
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy main restricted
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy-updates main restricted
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy universe
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy-updates universe
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy multiverse
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy-updates multiverse
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy-backports main restricted universe multiverse
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy-security main restricted
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy-security universe
+deb [arch=armhf] http://ports.ubuntu.com/ubuntu-ports jammy-security multiverse
+```
+After making changes, verify if the armhf architecture is still enabled in the package manager by re-running the command from Step 4.
+
+:::
+
+5. Update the package info: `sudo apt-get update && sudo apt-get upgrade -y`.
+6. Install webkitgtk for ARM: `sudo apt install libwebkit2gtk-4.0-dev:armhf`.
+7. Set the PKG_CONFIG_SYSROOT_DIR for the appropriate arch: `export PKG_CONFIG_SYSROOT_DIR=/usr/arm-linux-gnueabihf/`.
+8. Build the app by specifying the desired ARM version: `cargo tauri build --target armv7-unknown-linux-gnueabihf`.
+
+### Final Notes
+
+- The static website can be found in the `out` folder.
+- Rust may try to run your app after compiling it, which will fail on non-ARM platforms. Ignore this error, as the .deb package will be correctly built.
```

Comment:
This section can be removed the. The `out` folder is completely relevant and for the second point see my comment on the other Final Notes section

Reply:
I added the same warning twice. Once in the Prerequisites section and once at the start of the Action

Reply:
works for me for now 👍